main.js

```diff
@@ -0,0 +1,30 @@
+const next = document.querySelector('.next');
+const prev = document.querySelector('.prev');
+const slides = document.querySelector('.container')
+
+
+
+function addDisable() {
```

Comment:
Great fiunction, but it not only add disable, remove too.
use `changeDisable`, `manageDisable` etc.